src/components/WwwFrame/SearchBar.vue

```diff
@@ -233,7 +232,14 @@ export default {
 				} else {
 					query = { queryString: suggestion };
 				}
-				this.$router.push({ path: '/lend', query });
+
+				// Fallback to legacy filter if there's an unsupported query param
+				let filterUrl = '/lend/filter';
+				if (hasExcludedQueryParams(query)) {
+					filterUrl = '/lend';
+				}
+
+				window.location.href = `${filterUrl}?${new URLSearchParams(query).toString()}`;
```

Comment:
Can we please use the `$router` here so we don't force a page refresh?

Reply:
Sure, it'll just require some more logic here. I originally used the router, but there was extra searchbar behavior that had to be taken into account. The old app does a full page load, for example.

Reply:
Ok, I'm also wondering how many cases hit the `suggestion.url` case and if we need to augment that url before the window location.

The in page nav would only be an option for Ui so if it's too much muck and stuff we can skip for now. CPS and monolith would still have to do full page refreshes.

Reply:
There are currently no suggestions with `url`. Will there ever be? Can we remove it? I don't know the history of that prop.

Reply:
Pushed in page navigation for searchbar. Kept the full page load for country list, since we'd have to have prop drill the close method from the lend menu, and I didn't think that was worth the extra complication.

Reply:
I'm guessing it's a special case for certain items but also don't know offhand.

Reply:
Okay, I'll leave as is for now, since I don't have an example of what a suggestion `url` looks like.